core-tests/src/test/java/org/mule/runtime/core/privileged/processor/MessageProcessorsTestCase.java

```diff
@@ -150,7 +152,9 @@ public void processToApplyMap() throws Exception {
 
   @Test
   public void processToApplyMapInChain() throws Exception {
-    assertThat(processToApply(input, createChain(map)), is(output));
+    Processor chain = createChain(map);
+    assertThat(processToApply(input, chain), is(output));
+    disposeChain(chain);
```

Comment:
make this in a finally block

Reply:
or make chain into an attribute and dispose it an `@After` method

Reply:
Done.